### ISSUE TITLE
ghr does not work

### ISSUE THREAD

**Issue by dominikbraun** [tcnksm/ghr#122]
Hi,

in my CI, I'm running

```
ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} v0.1.9 artifacts/
```

to create my GH release using artifacts from the `artifacts` directory. Unfortunately, the following error occurs:

```
Invalid argument: you must set TAG and PATH name.
```

Any ideas?

**Comment by drazisil** [comment]
I think it should be:
```
ghr -t $GITHUB_TOKEN -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME -c $CIRCLE_SHA1 v0.1.9 artifacts/
```

**Comment by dominikbraun** [comment]
Unfortunately, that doesn't work:

![image](https://user-images.githubusercontent.com/36575275/73338199-5a7df600-4276-11ea-816d-2d81b37fba38.png)

**Comment by drazisil** [comment]
This wording was changed in https://github.com/tcnksm/ghr/commit/18e905231419757c57559f9d0901d6657700c3cb, it's possible this issue was fixed as well. Are you on the latest version?

**Comment by dominikbraun** [comment]
Seems like this is not the latest version. I'm currently using the CircleCI image `cibuilds/github:0.10` which comes with ghr... I'll try to download and run ghr manually.

**Comment by dominikbraun** [comment]
So now I get the new error message, but the error itself remains.

```
Invalid number of arguments: you must set a git TAG and optionally a PATH.
```

**Comment by drazisil** [comment]
I think i might see the issue in the code, but I'm not part of the team. Once i get a chance to try this I'll see if i get the same error.

**Comment by dominikbraun** [comment]
What piece of code do you have in mind?

**Comment by drazisil** [comment]
Line 70 of cli.go. i think its counting your flagged params, (token,  etc.) In the count. The code appears to fail now if its not exactly 1 or 2 params, the way im looking at it

**Comment by d4r1091** [comment]
Still having this issue...
Environment variable evaluated correclty setting: X_Y_Z
```
ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION_NUMBER} ~/project/output/packages
```

**Comment by engineerakki** [comment]
I am also facing the same issue.
Did anyone find a fix ?

**Comment by dominikbraun** [comment]
Yes: The error occured because one of the flags (e. g. one of the environment variables) was empty, and ghr thought the path argument would be missing... yes, pretty weird.

I could fix the error by quoting the environment variables:

```
ghr -t "${GITHUB_TOKEN}" \
      -u "${CIRCLE_PROJECT_USERNAME}" \
      -r "${CIRCLE_PROJECT_REPONAME}" \
      -c "${CIRCLE_SHA1}" \
      v0.1.9 artifacts/
```

That showed me that I provided an empty value for a certain flag, which I fixed then.

**Comment by antony-jr** [comment]
This also happens if you give wildcard for the path field.. That is ```/output/*``` will not be recognized, it should be ```/output/``` simply. This annoyed me a lot to be honest.

**Comment by jubr** [comment]
Ran into this as well, it looks like `CIRCLE_PROJECT_REPONAME` is not defined in some cases? Perhaps this happens with a free CircleCI instance?
